README.rst

```diff
@@ -1,61 +1,66 @@
+.. image:: https://img.shields.io/pypi/pyversions/sagemaker-inference.svg
+   :target: https://pypi.python.org/pypi/sagemaker-inference
+   :alt: Supported Python Versions
 
-This library is the serving subset of the `SageMaker Containers library <https://github.com/aws/sagemaker-containers>`__.
-
-Currently, this library is used by the following containers:
+.. image:: https://img.shields.io/badge/code_style-black-000000.svg
+   :target: https://github.com/python/black
+   :alt: Code Style: Black
 
-- `SageMaker MXNet Serving Container <https://github.com/aws/sagemaker-mxnet-serving-container>`__
+Serve machine learning models within a Docker container using Amazon SageMaker.
 
 -----------------
 Table of Contents
 -----------------
 .. contents::
     :local:
 
-Getting Started
----------------
+Background
+----------
 
-The main purpose of this library is to start up a model server within a container to enable serving on SageMaker.
+`Amazon SageMaker <https://aws.amazon.com/sagemaker/>`__ is a fully managed service that provides every developer and data scientist with the ability to build, train, and `deploy <https://aws.amazon.com/sagemaker/deploy/>`__ machine learning models quickly.
+SageMaker removes the heavy lifting from each step of the machine learning process to make it easier to develop and deploy high quality models.
 
-This library assumes the following `SageMaker inference requirements <https://docs.aws.amazon.com/sagemaker/latest/dg/your-algorithms-inference-code.html>`__ are met.
+Once you've trained a machine learning model, you can deploy it to a `Docker container <https://www.docker.com/resources/what-container>`__, where you can run your own inference code.
+The code that runs in a container is effectively isolated from its surroundings, ensuring a consistent runtime, regardless of where the container is deployed.
+By using a container, you can deploy machine learning models quickly and reliably at any scale.
+Additionally, you will be able to use your container to create a `Multi-Model Endpoint <https://docs.aws.amazon.com/sagemaker/latest/dg/multi-model-endpoints.html>`__, which enables you to deploy multiple models on a single endpoint and serve them using a single container.
 
-The following code block shows how to start the model server.
+The **SageMaker Inference Toolkit** library allows you to implement the model serving stack in your Docker container, making it compatible with SageMaker Multi Model Endpoints.
+This library's serving stack is built on `Multi Model Server (MMS) <https://github.com/awslabs/mxnet-model-server>`_, and it can serve your own models or those you trained on SageMaker using `any machine learning framework <https://docs.aws.amazon.com/sagemaker/latest/dg/frameworks.html>`__.
+(It is used in some of the `prebuilt SageMaker Docker images for inference <https://docs.aws.amazon.com/sagemaker/latest/dg/pre-built-containers-frameworks-deep-learning.html>`__.)
```

Comment:
I think this sentence is important enough to not be a parenthetical - it shows that we use this toolkit regularly on SageMaker. 😄